<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>Copyright(C) 2021 Deep.I Inc.

개발자 : deepi.contact.us@gmail.com 
라이센스 : MIT 라이센스
최신 버전은 블로그 링크를 참조해주세요.
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_tis">
       <property name="maximumSize">
        <size>
         <width>855</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>티스토리 블로그</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_git">
       <property name="text">
        <string>깃허브 저장소</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_close">
       <property name="text">
        <string>확인</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
